docs/user-guide/vsphere-machinesets.md

```diff
@@ -0,0 +1,951 @@
+
+| Placeholder | Meaning | Where to Find It |
+|------------|---------|------------------|
+| **`<INFRA_ID>`** | The unique infrastructure ID generated by OpenShift for your cluster. It is used to name MachineSets, Machines, and other cloud resources. | Run:<br>```oc get infrastructure cluster -o jsonpath='{.status.infrastructureName}'```<br>Example: `mycluster-abc12` |
+| **`<PORTGROUP_NAME>`** | The vSphere network (portgroup) the VM should attach to. Machine API will attach the cloned VM to this network, and RHCOS networking will configure itself (DHCP or Ignition). | In vCenter: **Networking → Distributed Port Groups**<br>Examples: `VM Network`, `ci-vlan-1221`, `Prod-Network-01` |
+| **`<RHCOS_VERSION>`** | The version of RHCOS you imported as an OVA template. This appears only in the template name you created. | Whatever you named the template, e.g.:<br>`rhcos-4.20.0` |
+| **`DATA_CENTER`** | The vSphere Datacenter containing the cluster’s VMs, templates, datastore, and resources. | vSphere UI → Inventory browser.<br>Same value appears under:<br>`Infrastructure.spec.platformSpec.vsphere.failureDomains[*].topology.datacenter` |
+| **`DATASTORE_NAME`** | The name of the datastore where new worker VMs will be created. **Must be the short name**, not the full inventory path. | vSphere UI → Datastores view.<br>Also appears in `Infrastructure` object under `.spec.platformSpec.vsphere.failureDomains[*].topology.datastore` (as a full path-use just the datastore name part) |
+| **`VCENTER.EXAMPLE.COM`** | The fully-qualified domain name of your vCenter server. | URL used to log into vSphere.<br>Also under:<br>`Infrastructure.spec.platformSpec.vsphere.vcenters[*].server` |
+| **`/DATA_CENTER/vm/path/to/template/...`** | The **inventory path** in vSphere to the folder containing your RHCOS template. Machine API must be able to find the template at this exact path. | Right-click the template in vSphere → **Copy Inventory Path** |
+| **`/DATA_CENTER/host/CLUSTER_NAME/Resources`** | The vSphere Resource Pool in which the new VMs will run. If you use a cluster, it's usually `/DATACENTER/host/<Cluster>/Resources`. | vSphere → Hosts & Clusters → right-click your cluster → **Resource Pool** tab |
+| **`/DATA_CENTER/vm/path/to/cluster/folder`** | The folder where cloned worker VMs should be placed. This controls where the VMs appear in the vCenter UI. | vSphere → Inventory browser → copy folder inventory path |
+| **`<CLUSTER_NAME>`** | The name of the vSphere cluster containing your ESXi hosts. Used inside the resource pool path. | vSphere → Hosts & Clusters → left tree view |
+
+---
+
+Apply:
+
+```bash
+oc apply -f machineset-vsphere-worker.yaml
+```
+
+---
+
+## 6. Wait for node to join and approve CSRs
+
+After applying the MachineSet, the Machine API will clone a new VM from the RHCOS template and boot it. The node will then request to join the cluster.
```

Comment:
Do users _need_ to deploy a new worker to set up platform integration? Would it be better to create the machineset with 0 replicas and scale up only when a worker is needed?